wiki/About/de.md

```diff
@@ -0,0 +1,5 @@
+# Über
+
+osu! wiki ist ein von der Gemeinschaft betriebenes und verfasstes Projekt um die osu! Nutzer das Spiel besser verstehen zu lassen.
```

Comment:
1. Wenn "osu! wiki" ein Eigenname sein soll (glaub ja schon?), dann würd ich das in Anführungszeichen packen. Ansonsten eher mit Artikel als ohne...
2. osu!-Nutzer
3. Gemeinschaft? Ich würde fast Community stehen lassen...
4. Irgendwie gefällt mir der Nebensatz nicht.. "um die Nutzer das Spiel besser verstehen zu lassen." Um das Spiel zu erklären? Und stimmt das überhaupt 100%ig? Es geht ja auch irgendwie nicht _nur_ um das Spiel direkt, oder? Kann man drüber reden, würde ich sagen